runtime/jcl/common/com_ibm_jvm_Stats.c

```diff
@@ -78,11 +80,13 @@ Java_com_ibm_jvm_Stats_getStats(JNIEnv *env, jobject obj)
 		softmx = max;
 	}
 
-	/* Calculation of free physical memory from existing function in mgmtosext.c*/
-	free = Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getFreePhysicalMemorySizeImpl(env, obj);
 
-	/* Calculation of total physical memory from port lib api */
-	tot = (jlong) j9sysinfo_get_physical_memory();
+	rc = j9sysinfo_get_memory_info(&memInfo);
```

Comment:
Looking at the AIX impl of this, there may be a bug. perfstat_memory_total() is documented as returning values in units of 4 KB pages, but the code uses `sysconf(_SC_PAGE_SIZE)` to calculate the availPhysical.

Reply:
omrvmem_get_available_physical_memory() seems implemented for all AIX, while retrieveAIXMemoryStats() is only implemented for !defined(J9OS_I5)